### ISSUE TITLE
using otel-collector-chart without hostNetwork

### ISSUE THREAD

**Issue by seekermarcel** [signalfx/splunk-otel-collector-chart#1014]
We want to use the otel-collector-chart with istio and thereby without the hostNetwork on a Linux system.
Currently the code only allows that for windows-based systems:
```yaml
...
  spec:
    {{- if not .Values.isWindows }}
    hostNetwork: true
    {{- end }}
...
```

Could there be another configurable option added to deactivate hostNetwork manually via the values.yaml?

**Comment by atoulme** [comment]
Please open a Splunk idea: https://ideas.splunk.com. Thanks!

**Comment by teimyBr** [comment]
https://github.com/splunk/splunk-connect-for-kubernetes is obsolete end of the year. 

We have to move to this. 

In the old one is was disabled by default. 

Now this is enabled. 

So we need the option to disable it again.

**Comment by atoulme** [comment]
Here, I logged it a while ago actually. Please upvote: https://ideas.splunk.com/ideas/EID-I-2101

**Comment by atoulme** [comment]
Please follow up on the Splunk idea. Thanks!

**Comment by theBNT** [comment]
For anyone reading here but not in the splunk idea (e.g. no splunkaccount), this has been addressed with PR #1107 and should land in the next release.

Thanks so much!
